app/uk/gov/hmrc/agentclientrelationships/model/MtdItIdBusinessDetails.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentclientrelationships.model
+import play.api.libs.json.Json
+import uk.gov.hmrc.agentmtdidentifiers.model.MtdItId
+
+case class MtdItIdBusinessDetails(mtdbsa: MtdItId)
```

Comment:
i dont think we need a separate case class for wrapping mtdId ?

Reply:
same reason but can rearrange